_data/communication.yml

```diff
@@ -179,6 +179,14 @@ websites:
       software: Yes
       doc: https://help.sendloop.com/docs/securing-your-account
 
+    - name: Signal
+      url: https://www.signal.org/
+      img: signal.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
```

Comment:
In my experience, you cannot verify your number by phone call, only SMS.

Reply:
https://support.signal.org/hc/en-us/articles/115005335227

Reply:
Thanks, @stephengroat